src/testHelpers.ts

```diff
@@ -179,3 +179,34 @@ export async function createRollupHelper({
     createRollupInformation,
   };
 }
+
+export function testHelper_createCustomParentChain(params?: { id?: number }) {
```

Comment:
opted in to use a `testHelper_` prefix because it's easier to differentiate what's a test helper and what's actual production code when writing a test case